packages/system/package.json

```diff
@@ -9,15 +9,18 @@
   "dependencies": {
     "@emotion/core": "10.1.1",
     "@theme-ui/css": "0.3.3",
+    "emotion": "10.0.27",
     "lodash.pick": "4.4.0"
   },
   "peerDependencies": {
     "react": "^16.x",
     "react-dom": "^16.x"
   },
   "devDependencies": {
+    "@theme-ui/css": "0.3.1",
```

Comment:
In unserer Repo ist Version 0.3.3
Du setzt sie hier runter, ist wahrscheinlich wegen depandabot updates nicht aktuell.

Reply:
In der derzeitigen beta sind die TS styles drin -> 0.4.1 oder so

PS: warum ist das eine `devDependency` 🤔